docs/gestor_tareas.md

```diff
@@ -0,0 +1,69 @@
+# Gestor de Tareas
+La automatización de tareas es fundamental para optimizar los flujos de trabajo en el desarrollo de software. Este documento compara las herramientas más relevantes para seleccionar la más adecuada para proyectos en Java.
+
+## Criterios de selección
+Para elegir el gestor de tareas para nuestro proyecto en Java, hemos considerado los siguientes criterios:
+
+Facilidad de Configuración: Tiempo promedio necesario para configurar tareas básicas (minutos).
+- *Eficiencia*: Tiempo de ejecución de tareas comunes en proyectos estándar (ms).
```

Comment:
De veras, ¿por qué ponéis esto? ¿En serio creéis que la ejecución de la tarea va a tener importancia con respecto al tiempo que tarde la tarea en sí?